src/main/openshift/production.yml

```diff
@@ -0,0 +1,64 @@
+apiVersion: apps/v1
```

Comment:
I created the openshift configs in  src/main/openshift since this is where you added them in the projects-bots-api repo:
https://github.com/EclipseFdn/projects-bots-api/tree/master/src/main/openshift

Please advise if we should move them ./k8s as we do with hugo deployments.

Also, could you explain to me how openshift finds these config files in a project?

Reply:
> Please advise if we should move them into ./k8s as we do with hugo deployments.

src/main/ is usually where all sources go in a maven project, with a subfolder per type of sources (src/main/java for java sources, src/main/docker for dockerfiles...). We can either use src/main/openshift or src/main/k8s (may be better to stay closer to what we have in hugo deployments... when I initially wrote the projects-bots-api, I did not write the hugo deployments files yet).

For hugo deployment, we don't have the maven folders structure, so putting the k8s at the root of the repo looks good enough to me.

> Also, could you explain to me how openshift finds these config files in a project?

Openshift does not find these config files. The .yaml files are only applied by webmaster once to create the deployment/service/route in the clutser. Successful builds on master then just update the container images of the deployment, it never re-apply those file. If changes need to be made and applied to those files, you will need to ask an admin to apply those.

Reply:
+1 I will rename openshift folder to k8s for consistency.